src/pages/login.js

```diff
@@ -1,61 +1,74 @@
+import { Button, Flex, Input, Heading, Text, jsx } from 'theme-ui'
+import { toast } from 'react-toastify';
 
-import { AiFillGoogleCircle } from 'react-icons/ai'
+import Layout from '../components/layout'
+import PageHeader from '../components/pageheader'
+import Container from '../components/container'
 
-import firebase from 'firebase/app'
-import useFirebase from '../firebase'
-import useAuth from '../util/auth'
-/** @jsx jsx */
+import getFirebase from '../firebase'
+
+const Login = ({ data }) => {
+  const firebaseApp = getFirebase()
 
-export default function login() {
-  const firebaseApp = useFirebase()
-  const [error, setError] = useState(null)
+  const [email, setEmail] = useState('')
 
-  const handleGoogleLogin = async () => {
+  const onSubmit = async e => {
+    e.preventDefault()
     if (!firebaseApp) return
-    const provider = new firebase.auth.GoogleAuthProvider()
-    try {
-      await firebaseApp.auth().signInWithPopup(provider)
-    } catch (e) {
-      setError(e.message)
+
+    if (!/\S+@\S+\.\S+/.test(email)) {
+      toast.error('Invalid email.')
       return
     }
-  }
 
-  const user = useAuth()
-  if (user !== null) {
-    return <Redirect to='/' noThrow />
+    /* TODO: use toasts? */
+    try {
+      await firebaseApp.firestore().collection('reminders').add({ email })
+      toast.success('Submitted!')
+      setEmail('')
+    } catch (e) {
+      toast.error('Something went wrong. Let us know at hello@climatedu.org.')
+    }
   }
 
   return (
-    <Box sx={{ height: '100vh', display: 'grid' }}>
-      <Box
-        sx={{ margin: 'auto', width: '100%', maxWidth: 300, display: 'grid' }}
-      >
+    <Layout>
+      <PageHeader
+        primary='Login'
+        secondary="We're still building this course. Come back and join us soon!"
+      />
+      <Container>
         <Heading
-          as='h1'
-          sx={{ textAlign: 'center', color: 'primary', marginBottom: 20 }}
+          sx={{
+            textAlign: 'center',
+            mb: 3,
+          }}
         >
-          Log In
+          Leave your email, and we&apos;ll remind you:
         </Heading>
-        <Button onClick={handleGoogleLogin} sx={{ cursor: 'pointer' }}>
-          Sign in with Google
-          <AiFillGoogleCircle
-            size='1.5em'
-            sx={{
-              color: 'primary',
-              verticalAlign: 'middle',
-              marginLeft: 10,
-            }}
+        <Flex
+          as='form'
+          sx={{
+            maxWidth: 300,
+            m: 'auto',
+            flexDirection: 'column',
+          }}
+        >
+          <Input
+            aria-label='Email'
+            value={email}
+            onChange={e => setEmail(e.target.value)}
+            sx={{ mb: 2 }}
           />
-        </Button>
-        <Text sx={{ color: 'highlight', marginTop: 20, textAlign: 'center' }}>
-          {error}
-        </Text>
-      </Box>
-    </Box>
+          <Button onClick={onSubmit}>
```

Comment:
The event handler should be attached to the form submit event, not the click event on the button. This change will improve a11y and keyboard usability. The same issue was previously brought up [here](https://github.com/climatEDU/climatEDU/pull/22#discussion_r485228624).